upstream-sha256: 67b1b9006012201ebc6a1bf81e315da77a923bd36d18d859dc1943af1f3618d3
actual-sha256: 07411603f46dbaa782c9ff172de6a1d748127a15e171e822d00a08fa48aba305
--- upstream.expected
+++ tolk.actual
@@ -8,15 +8,6 @@
     float val5 = (*(data1_128+(Lidx1+80)));
     float val6 = (*(data1_128+(Lidx1+96)));
     float val7 = (*(data1_128+Lidx1));
-    *(data0_16+Lidx1) = (val7+val1+val2+val3+val4+val5+val6+val0);
+    *(data0_16+Lidx1) = (0.0f+val7+val1+val2+val3+val4+val5+val6+val0);
   }
 }
----
-typedef float float4 __attribute__((aligned(16),ext_vector_type(4)));
-void E_4_4(float* restrict data0_16, float* restrict data1_16) {
-  for (int Lidx0 = 0; Lidx0 < 4; Lidx0++) {
-    int alu0 = (Lidx0<<2);
-    float4 val0 = (*((float4*)((data1_16+alu0))));
-    *((float4*)((data0_16+alu0))) = val0;
-  }
-}
